facebook/build.gradle

```
@@ -35,7 +35,7 @@ android {
 }
 
 dependencies {
-    api "com.facebook.android:facebook-login:[13.2.0, 14.0["
+    api "com.facebook.android:facebook-login:[13.2.0, 14.1.1]"
```

The upper limit is set because the SDK has not been tested for compatibility with higher versions. If we know it's compatible with Facebook SDK 14.x we can set the upper range to <=15. If we know it's compatible with Facebook SDK 15 and 16 we can even set it to <=17.

---

I'm able to test the Facebook Login feature only, is that sufficient to say it's compatible? BTW last published version is 16.0.0 https://github.com/facebook/facebook-android-sdk/commits/main

I can try with this and let you know:
```suggestion
    api "com.facebook.android:facebook-login:[13.2.0, 16.0.0]"
```

---

@mtrezza tested Facebook Login with 16.0.0 and it's working.

---

Great; coupd you also test with version 15? Then we can set the upper bount to <17.

---

@mtrezza I already set the upper bound to <17
Version 14 was already tested OK as this PR was initially using this one
Version 15 has just been tested OK

---

Great, so I think you could change this to 
`api "com.facebook.android:facebook-login:[13.2.0, 17.0.0["`, which means SDK is supported up to excluding version 17.0.0

---

@mtrezza please check the commits, I've done this yesterday